gammapy/data/data_store.py

```diff
@@ -326,6 +326,8 @@ def copy_obs(self, obs_id, outdir, verbose=False, clobber=False):
             List of observations to copy
         outdir : str, Path
             Directory for the new store
+        hdu_class : list
```

Comment:
This PR looks good and tests passed.

One suggestion ... I'm not sure how to make the docs better, maybe put "list of str" here, or maybe even add a link to `gammapy.data.HDUIndexTable.VALID_HDU_CLASS` so that the user knows what they can select?